test/nl/surf/eduhub_rio_mapper/e2e_test.clj

```diff
@@ -0,0 +1,124 @@
+(ns nl.surf.eduhub-rio-mapper.e2e-test
+  (:require [clojure.test :refer :all]
+            [nl.surf.eduhub-rio-mapper.e2e-helper :refer :all]
+            [nl.surf.eduhub-rio-mapper.remote-entities-helper :refer [remote-entities-fixture]])
+  (:import (java.util UUID)))
+
+(use-fixtures :once with-running-mapper remote-entities-fixture)
+
+(def test-eigensleutel (str (UUID/randomUUID)))
+(def parent-rio-code-atom (atom nil))
+(def child-rio-code-atom (atom nil))
+
+(deftest ^:e2e create-edspecs
+  (testing "scenario [1a]: Test /job/dry-run to see the difference between the edspec parent in OOAPI en de opleidingeenheid in RIO. You can expect RIO to be empty, when you start fresh."
+
+    (let [job (post-job :dry-run/upsert :education-specifications
+                        (ooapi "education-specifications/parent"))]
+      (is (= "done" (job-result-status job)))
+      (is (= "not-found" (:status (job-result-attributes job))))))
+
+  #_ ;; TODO this fails because eduspec is added to rio but linking currently silently fails
+  (testing "scenario [4b]: Test /job/upsert with the program. You can expect an error, because the edspec child is not upserted."
+
```

Comment:
Is this an error in the description? There is nothing about programs in here. Also if the intent is to upsert an eduspec child I would think it should be

"You can expect an error, because the edspec **parent** is not upserted."